packages/pipeline/src/parsers/ohlcv_external/crypto_compare.ts

```diff
@@ -0,0 +1,42 @@
+import { CryptoCompareOHLCVRecord } from '../../data_sources/ohlcv_external/crypto_compare';
+import { OHLCVExternal } from '../../entities';
+
+const ONE_SECOND = 1000; // Crypto Compare uses timestamps in seconds instead of milliseconds
+
+export interface OHLCVMetadata {
+    exchange: string;
+    fromSymbol: string;
+    toSymbol: string;
+    source: string;
+    observedTimestamp: number;
+}
+/**
+ * Parses OHLCV records from Crypto Compare into an array of OHLCVExternal entities
+ * @param rawRecords an array of OHLCV records from Crypto Compare (not the full response)
+ */
+export function parseRecords(rawRecords: CryptoCompareOHLCVRecord[], metadata: OHLCVMetadata): OHLCVExternal[] {
+    if (rawRecords.length > 1) {
+        const intervalInSeconds = Math.abs(rawRecords[0].time - rawRecords[1].time);
```

Comment:
i wrote a utility function to convert current time to unix epoch time to interface w/ cc. since you don't need the utility function, can you delete it in this PR?

Reply:
@wakkadojo I looked around and deleted `getHourInUnixTime`